templates/juniper_junos_show_lacp_interfaces.textfsm

```diff
@@ -0,0 +1,19 @@
+Value Required AGGREGATED_INTERFACE (\S+)
+Value List MEMBER_INTERFACE (\S+)
+Value List RECEIVE_STATE (\S+\s?\w+)
+Value List TRANSMIT_STATE (\S+\s?periodic)
+Value List MUX_STATE (\S+\s?\S*)
+
+Start
+  ^Aggregated -> Continue.Record
+  ^Aggregated\sinterface:\s${AGGREGATED_INTERFACE}(\s|$$) -> Continue
+  ^\s+LACP\s+state: -> Continue
+  ^\s+LACP\sprotocol: -> LACP_PROTO
+  ^\s*$$
+  ^{master:\d+}
+
+LACP_PROTO
+  ^\s+${MEMBER_INTERFACE}\s+${RECEIVE_STATE}\s+${TRANSMIT_STATE}\s+${MUX_STATE}(\s|$$) -> Continue
+  ^\s*$$
+  ^Aggregated -> Continue.Record
+  ^Aggregated\sinterface:\s${AGGREGATED_INTERFACE}(\s|$$) -> Start
```

Comment:
Do you mind adding the `^. -> Error` to both states? And then we can get this merged in.

Reply:
I added `^. -> Error`, but tox result is failed:(
```
Start
  ^Aggregated -> Continue.Record
  ^Aggregated\sinterface:\s${AGGREGATED_INTERFACE}(\s|$$) -> Continue
  ^\s+LACP\s+state: -> Continue
  ^\s+LACP\sprotocol: -> LACP_PROTO
  ^\s*$$
  ^{master:\d+}
  ^. -> Error

LACP_PROTO
  ^\s+${MEMBER_INTERFACE}\s+${RECEIVE_STATE}\s+${TRANSMIT_STATE}\s+${MUX_STATE}(\s|$$) -> Continue
  ^\s*$$
  ^Aggregated -> Continue.Record
  ^Aggregated\sinterface:\s${AGGREGATED_INTERFACE}(\s|$$) -> Start
  ^. -> Error
```

Reply:
Here is a working one:

```
Value Required AGGREGATED_INTERFACE (\S+)
Value List MEMBER_INTERFACE (\S+)
Value List RECEIVE_STATE (\S+\s?\w+)
Value List TRANSMIT_STATE (\S+\s?periodic)
Value List MUX_STATE (\S+\s?\S*)

Start
  ^Aggregated -> Continue.Record
  ^Aggregated\s+interface:\s+${AGGREGATED_INTERFACE}
  ^\s+LACP\s+state
  ^\s+LACP\s+protocol -> LACP_PROTO
  ^\s+\S+
  ^{master:\d+}
  ^\s*$$
  ^. -> Error

LACP_PROTO
  ^\s+${MEMBER_INTERFACE}\s+${RECEIVE_STATE}\s+${TRANSMIT_STATE}\s+${MUX_STATE}
  ^Aggregated -> Continue.Record
  ^Aggregated\s+interface:\s+${AGGREGATED_INTERFACE} -> Start
  ^\s+LACP\s+state -> Start
  ^{master:\d+}
  ^\s*$$
  ^. -> Error
```

Reply:
Thank you. Finally I add `^.` and `^master`, test result is OK.
I was just a little bit confused to use `Continue` statement.